tests/Encoder/Issue238/IssueTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+declare(strict_types=1);
+
+namespace Neomerx\Tests\JsonApi\Encoder\Issue238;
+
+/**
+ * Copyright 2015-2019 info@neomerx.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use Neomerx\JsonApi\Encoder\Encoder;
+use Neomerx\Tests\JsonApi\BaseTestCase;
+use Neomerx\Tests\JsonApi\Data\Models\Author;
+use Neomerx\Tests\JsonApi\Data\Models\Post;
+use Neomerx\Tests\JsonApi\Data\Schemas\AuthorSchema;
+use Neomerx\Tests\JsonApi\Data\Schemas\PostSchema;
+
+/**
+ * @package Neomerx\Tests\JsonApi
+ */
+class IssueTest extends BaseTestCase
+{
+    public function testIterableParameterForWithIncludedPaths(): void
+    {
+        $author = Author::instance(238, 'Susan', 'Smith');
+        $post = Post::instance(11, 'Generators and Arrays', 'A tale of incompatible types', $author);
+
+        $encoder = Encoder::instance([
+            Author::class => AuthorSchema::class,
+            Post::class => PostSchema::class,
+        ]);
+
+        $encoder = $encoder->withIncludedPaths($this->generateIncludeList());
```

Comment:
Before the patch to `EncoderPropertiesTrait` this line would cause a `TypeError`.